src/content/en/fundamentals/accessibility/02-semantics-bultin/sembuiltin-six.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: shared/narrow
+title: "Text Alternatives for Images"
+description: "Using the alt attribute to provide text alternatives for images"
+published_on: 2016-03-01
+updated_on: 2016-03-01
+order: 6
+translation_priority: 0
+authors:
+  - megginkearney
+  - dgash
+key-takeaways:
+  tldr: 
+    - "Learn what accessibility means and how it applies to web development."
+    - "Learn how to make web sites accessible and usable for everyone."
+    - "Learn how to include basic accessibility with minimal development impace."
+    - "Learn what HTML features are available and how to use them to improve accessibility."
+    - "Learn about advanced accessibility techniques for creating polished accessibility experiences."
+notes:
+  efficiency:
+    - "Understanding the accessibility issue, its scope, and its impact can make you a better web developer."
+  problem-solving:
+    - "Catching, identifying, and removing potential accessibility roadblocks before they happen can improve your development process and reduce maintenance requirements."
+---
+
+Images are an important component of most web pages, and are of course a particular sticking point for low-vision users. We must consider the role an image plays in a page to work out what type of text alternative it should have. Take a look at this page.
+
+![funioncat1](imgs/funioncat1.png)
+
+In the page we have a picture of a cat, illustrating the article on cats' well-known judgemental behavior. ChromeVox announces this image using its literal name, "/160204193356-01-cat-500.jpg" (seen in the black voiceover bar). That's accurate, but not at all useful.
```

Comment:
We can also add a code snippet here.

`<img src="/160204193356-01-cat-500.jpg">`